<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JarvisUI</class>
 <widget class="QDialog" name="JarvisUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1401</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="bg_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1381</width>
     <height>871</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../GUI MATERIAL/B.G/Black_Template.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="bg_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>471</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(74, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bg_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>581</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(74, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Gif_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>441</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/KALPIT VORA/Desktop/5a9628b9256d9e54afd861edc949b0f2_w200.webp</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Gif_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>561</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../GUI MATERIAL/ExtraGui/61de57a229cd6901f83fe0f9d04b5dca_w200.webp</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Gif_3">
   <property name="geometry">
    <rect>
     <x>-100</x>
     <y>330</y>
     <width>661</width>
     <height>431</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../GUI MATERIAL/VoiceReg/__1.gif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="bg_5">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>30</y>
     <width>261</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(74, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Gif_4">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>40</y>
     <width>241</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../GUI MATERIAL/ExtraGui/Earth.gif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="bg_6">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>620</y>
     <width>291</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../GUI MATERIAL/B.G/gyhf.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Texttime_2">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>750</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: Transparent;
 border-radius:none;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Texttime_1">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>630</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: Transparent;
 border-radius:none;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Strat">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>210</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> 
background-color: rgb(0, 0, 0);
font: 20pt &quot;MS Shell Dlg 2&quot;;
border-radius:none;
color: rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QLabel" name="bg_7">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>200</y>
     <width>261</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(74, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Exit">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>290</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> 
background-color: rgb(0, 0, 0);
font: 20pt &quot;MS Shell Dlg 2&quot;;
border-radius:none;
color: rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
  <widget class="QLabel" name="bg_8">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>400</y>
     <width>291</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../GUI MATERIAL/B.G/gyhf.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Texttime_3">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>530</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: Transparent;
 border-radius:none;
</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Texttime_4">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>410</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: Transparent;
border-radius:none;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_yt">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>370</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 255);
font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Youtube</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton__cm">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>370</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 255);
font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Chrome</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_wp">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>370</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 255);
font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Whatsapp</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>590</y>
     <width>531</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../GUI MATERIAL/ExtraGui/initial.gif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>470</y>
     <width>461</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../GUI MATERIAL/ExtraGui/Jarvis_Gui (1).gif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>bg_1</zorder>
  <zorder>Gif_3</zorder>
  <zorder>bg_2</zorder>
  <zorder>bg_3</zorder>
  <zorder>Gif_1</zorder>
  <zorder>Gif_2</zorder>
  <zorder>bg_5</zorder>
  <zorder>Gif_4</zorder>
  <zorder>bg_6</zorder>
  <zorder>Texttime_2</zorder>
  <zorder>Texttime_1</zorder>
  <zorder>bg_7</zorder>
  <zorder>pushButton_Strat</zorder>
  <zorder>pushButton_Exit</zorder>
  <zorder>bg_8</zorder>
  <zorder>Texttime_3</zorder>
  <zorder>Texttime_4</zorder>
  <zorder>pushButton_yt</zorder>
  <zorder>pushButton__cm</zorder>
  <zorder>pushButton_wp</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
